src/wt/main.js

```diff
@@ -1,5 +1,36 @@
+// main.js
+import {Worker} from "worker_threads";
+import os from "os";
+
 const performCalculations = async () => {
-    // Write your code here
+  const numThreads = os.cpus().length;
+  const workers = [];
+  const results = [];
+
+  for (let i = 0; i < numThreads; i++) {
+    const worker = new Worker('./src/wt/worker.js');
```

Comment:
It doesn't work. We need to use the path to the worker module instead of the URL.

Reply:
UPD: it works. But only from the root directory.